src/3d/castletransform_physics.inc

```diff
@@ -720,6 +721,7 @@ procedure TRigidBody.InitializeTransform(const Transform: TCastleTransform);
     FKraftBody.Finish;
 
     UpdateCollides(Transform);
+    UpdateExist(Transform.GetExistsRecursively);
```

Comment:
Ideally `TRigidBody` should not call a recursive function here, but only access a ready field `Transform.ExistsRecursively`. Disclaimer: I did not analyze if this is feasible to implement :)